module/observations/helper.js

```diff
@@ -2255,6 +2255,48 @@ module.exports = class ObservationsHelper {
         })
     }
 
+
+ /**
+      * Get imported observation documents.
+      * @method
+      * @name getImportedObservations
+      * @param  {String} userId - userId of user.
+      * @param  {String} programId - program Id.
+      * @returns {result} - all the observation which user has started in that program. 
+     */
+
+    static getImportedObservations(userId, programId){
+        return new Promise(async (resolve, reject) => {
+            try {
+                
+                let surveyData = await this.observationDocuments({
+                    programId : programId,
+                    createdBy : userId,
```

Comment:
@ankitpws the condition sequence has to be reversed, createdBy and then programId, it makes the query faster

Reply:
done